Source/Painting/SvgLinearGradientServer.Drawing.cs

```diff
@@ -118,7 +118,11 @@ private GradientPoints ExpandGradient(ISvgBoundable boundable, PointF specifiedS
             var effectiveEnd = specifiedEnd;
             var intersectionPoints = CandidateIntersections(bounds, specifiedStart, specifiedEnd);
 
-            Debug.Assert(intersectionPoints.Count == 2, "Unanticipated number of intersection points");
+            if (intersectionPoints.Count < 2)
+            {
+                Console.WriteLine("Unanticipated number of intersection points!");
```

Comment:
If this condition is not Error, there is no need to print to  console.

Reply:
Well, it's a similar edge case as 10 lines above. Instead of removing the Console.WriteLine we could also Debug.Fail, but this would be a bit too strict in my opinion. I think it's noteworthy that we omit the rending of a part of the SVG instructions.
For now, I removed the line as you suggested.

Reply:
There is no issue with SVG specifications, right ?